modules/administration-guide/pages/customizing-openshift-che-consolelink.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: PROCEDURE
+:description: Customizing OpenShift Eclipse Che ConsoleLink
+:keywords: administration guide, customizing, consolelink
+:navtitle: Customizing OpenShift Eclipse Che ConsoleLink
+:page-aliases: installation-guide:customizing-openshift-che-consolelink.adoc
+
+[id="customizing-openshift-che-consolelink"]
+= Customizing {ocp} {prod} ConsoleLink
```

Comment:
@tolusha just wondering if Red Hap Application dropdown menu is also using ConsoleLink object or is it called smth. differently?

Reply:
Sorry, which dropdown menu we are talking about?

Reply:
<img width="377" alt="Screenshot 2023-11-17 at 15 41 16" src="https://github.com/eclipse/che/assets/1461122/df3e86a7-326a-45ed-b365-21a2d84f28df">

Reply:
Red Hap Application dropdown menu is based on ConsoleLink objects